modules/contact/src/constraints/NormalNodalLMMechanicalContact.C

```diff
@@ -46,10 +46,22 @@ NormalNodalLMMechanicalContact::NormalNodalLMMechanicalContact(const InputParame
     _disp_z_id(coupled("disp_z")),
     _c(getParam<Real>("c")),
     _epsilon(std::numeric_limits<Real>::epsilon()),
-    _ncp_type(getParam<MooseEnum>("ncp_function_type"))
-
+    _ncp_type(getParam<MooseEnum>("ncp_function_type")),
+    _fe_problem(*getCheckedPointerParam<FEProblem *>("_fe_problem", "Must be using FEProblem.")),
```

Comment:
`FEProblem` --> `FEProblemBase`?

Reply:
Yeah, I think it should be `FEProblemBase`. It should be addressed now.